examples/1-advanced/08-classifier.py

```diff
@@ -0,0 +1,270 @@
+model, freezes its backbone, and trains a small multi-layer perceptron (MLP) on
+top of the extracted features for classification tasks.
+
+In this example, we will classify carbon allotropes (diamond, graphite, and graphene).
+
+Creating the Dataset
+--------------------
+
+First, we need to create a dataset with different carbon structures. We'll generate
+simple structures for diamond, graphite, and graphene, and label them with one-hot
+encoded class labels. The classifier also supports soft/fractional targets for cases
+where the class membership is uncertain.
+
+"""
+
+# %%
+
+import subprocess
+
+import ase
+import ase.io
+import chemiscope
+import matplotlib.pyplot as plt
+import numpy as np
+from ase.build import bulk, graphene
+from metatomic.torch import ModelOutput
+from metatomic.torch.ase_calculator import MetatomicCalculator
+
+
+# %%
+#
+# We generate structures for three carbon allotropes:
+# - Diamond (class 0)
+# - Graphite (class 1)
+# - Graphene (class 2)
+
+np.random.seed(42)
+
+structures = []
+
+# Generate 10 diamond structures with small random perturbations
+for i in range(10):
+    diamond = bulk("C", "diamond", a=3.57)
+    diamond = diamond * (2, 2, 2)  # Make it bigger
+    diamond.rattle(stdev=0.5, seed=i)  # Add random perturbations
+    # One-hot encoding for diamond (class 0): [1.0, 0.0, 0.0]
+    diamond.info["class"] = [1.0, 0.0, 0.0]
+    structures.append(diamond)
+
+# Generate 10 graphite structures (using layered graphene-like structures)
+for i in range(10):
+    # Create a graphite-like structure
+    graphite = graphene(formula="C2", size=(3, 3, 1), a=2.46, vacuum=None)
+    # Stack two layers
+    layer2 = graphite.copy()
+    layer2.translate([0, 0, 3.35])
+    graphite.extend(layer2)
+    graphite.set_cell([graphite.cell[0], graphite.cell[1], [0, 0, 6.7]])
+    graphite.rattle(stdev=0.5, seed=i)
+    # One-hot encoding for graphite (class 1): [0.0, 1.0, 0.0]
+    graphite.info["class"] = [0.0, 1.0, 0.0]
+    structures.append(graphite)
+
+# Generate 10 graphene structures (single layer)
+for i in range(10):
+    graphene_struct = graphene(formula="C2", size=(3, 3, 1), a=2.46, vacuum=10.0)
+    graphene_struct.rattle(stdev=0.5, seed=i)
+    # One-hot encoding for graphene (class 2): [0.0, 0.0, 1.0]
+    graphene_struct.info["class"] = [0.0, 0.0, 1.0]
+    structures.append(graphene_struct)
+
+# Save the structures to a file (these will be used for training)
+ase.io.write("carbon_allotropes.xyz", structures)
+
+# %%
+#
+# Getting a pre-trained universal model
+# -------------------------------------
+#
+# Here, we download a pre-trained model checkpoint that will serve as the backbone
+# for our classifier. We will use PET-MAD, a universal interatomic potential for
+# materials and molecules.
+
+subprocess.run(
+    [
+        "wget",
+        "https://huggingface.co/lab-cosmo/pet-mad/resolve/v1.0.2/models/pet-mad-v1.0.2.ckpt",
+    ],
+    check=True,
+)
```

Comment:
```suggestion
PET_MAD_URL = "https://huggingface.co/lab-cosmo/pet-mad/resolve/v1.0.2/models/pet-mad-v1.0.2.ckpt"

subprocess.run(["wget", PET_MAD_URL], check=True)
```

Also, do we want to use a more recent version of pet-mad?